README.md

```diff
@@ -114,6 +114,9 @@ This project uses Nix to manage the project's environment. If you'd like to buil
 # Install javascript dependencies
 just install-deps
 
+# Translate schemas into Elm code
+just translate-schemas
```

Comment:
This is actually not necessary, the `dev-build` command does this for you. Which is also run by default when doing `just`. We should however add it to the `staging-build` and `production-build` commands as well (in case you never do a dev build).

Reply:
Right, that makes sense. Pushed a commit that removes it from the README and adds it to the staging and production builds.

Also, it was failing in `dev-build`. Moving it forward in the commands before `elm` got it working.

Reply:
Whoops 😅 Thanks for fixing that!